recipes/flit-core/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "flit-core" %}
```

Comment:
`flit-core` is provided by [`flit-feedstock`](https://github.com/conda-forge/flit-feedstock/blob/main/recipe/meta.yaml) multi-output recipe, which also demonstrates how `solora(-assets)` can be built in the same `meta.yaml`. All feedstocks -> packages can be seen on https://conda-forge.org/feedstock-outputs/